nmostesting/suites/IS1101Test.py

```diff
@@ -246,23 +248,23 @@ def test_03_05(self, test):
                 if not response.json()["edid_support"]:
                     self.not_edid_connected_outputs.append(response.json()["id"])
             else:
-                return test.FAIL(response.json())
+                return test.FAIL("The request has not succeeded.", response.json())
         if len(self.not_edid_connected_outputs) == 0:
-            return test.UNCLEAR("Outputs supporting edid")
+            return test.UNCLEAR("Outputs supporting edid.")
         return test.PASS()
 
     def test_03_05_01(self, test):
         """
         Verify that there is no EDID support.
         """
         if len(self.not_edid_connected_outputs) == 0:
-            return test.UNCLEAR("none of not edid connected outputs")
+            return test.UNCLEAR("None of not edid connected outputs.")
         for output_id in self.not_edid_connected_outputs:
             _, response = TestHelper.do_request(
                 "GET", self.compat_url + "outputs/" + output_id + "/edid/"
             )
             if response.status_code != 204:
-                return test.UNCLEAR("status code should be 204")
+                return test.UNCLEAR("Status code should be 204.")
```

Comment:
Why is this `UNCLEAR` (Could Not Test)? If `204` is the only acceptable response, shouldn't this be a `FAIL` (Fail)?